src/library/returnBook/ReturnBookUI.java

```diff
@@ -4,85 +4,85 @@
 
 public class ReturnBookUI {
 
-	public static enum uI_sTaTe { INITIALISED, READY, INSPECTING, COMPLETED };
+    public static enum UiState {INITIALISED, READY, INSPECTING, COMPLETED};
 
-	private rETURN_bOOK_cONTROL CoNtRoL;
-	private Scanner iNpUt;
-	private uI_sTaTe StATe;
+    private ReturnBookControl control;
+    private Scanner input;
+    private UiState state;
 
-	
-	public ReturnBookUI(rETURN_bOOK_cONTROL cOnTrOL) {
-		this.CoNtRoL = cOnTrOL;
-		iNpUt = new Scanner(System.in);
-		StATe = uI_sTaTe.INITIALISED;
-		cOnTrOL.sEt_uI(this);
-	}
+    
+    public ReturnBookUI(ReturnBookControl control) {
+        this.control = control;
+        input = new Scanner(System.in);
+        state = UiState.INITIALISED;
+        control.sEt_uI(this);
+    }
 
 
-	public void RuN() {		
-		oUtPuT("Return Book Use Case UI\n");
-		
-		while (true) {
-			
-			switch (StATe) {
-			
-			case INITIALISED:
-				break;
-				
-			case READY:
-				String BoOk_InPuT_StRiNg = iNpUt("Scan Book (<enter> completes): ");
-				if (BoOk_InPuT_StRiNg.length() == 0) 
-					CoNtRoL.sCaNnInG_cOmPlEtE();
-				
-				else {
-					try {
-						int Book_Id = Integer.valueOf(BoOk_InPuT_StRiNg).intValue();
-						CoNtRoL.bOoK_sCaNnEd(Book_Id);
-					}
-					catch (NumberFormatException e) {
-						oUtPuT("Invalid bookId");
-					}					
-				}
-				break;				
-				
-			case INSPECTING:
-				String AnS = iNpUt("Is book damaged? (Y/N): ");
-				boolean Is_DAmAgEd = false;
-				if (AnS.toUpperCase().equals("Y")) 					
-					Is_DAmAgEd = true;
-				
-				CoNtRoL.dIsChArGe_lOaN(Is_DAmAgEd);
-			
-			case COMPLETED:
-				oUtPuT("Return processing complete");
-				return;
-			
-			default:
-				oUtPuT("Unhandled state");
-				throw new RuntimeException("ReturnBookUI : unhandled state :" + StATe);			
-			}
-		}
-	}
+    public void run() {        
+        output("Return Book Use Case UI\n");
+        
+        while (true) {
+            
+            switch (state) {
+            
+            case INITIALISED:
+                break;
+                
+            case READY:
+                String bookInputString = input("Scan Book (<enter> completes): ");
+                if (bookInputString.length() == 0) {
+                    control.scanningComplete();
+                }
+                else {
+                    try {
+                        int Book_Id = Integer.valueOf(bookInputString).intValue();
+                        control.bookScanned(Book_Id);
+                    }
+                    catch (NumberFormatException e) {
+                        output("Invalid bookId");
+                    }                    
+                }
+                break;                
+                
+            case INSPECTING:
+                String AnS = input("Is book damaged? (Y/N): ");

```

Comment:
The string "AnS" should be all lowercase.

Reply:
Resolution: https://github.com/9Mcgusto6/ITC205-Team17/pull/5/commits/c4f1ae1fd608faa484f4ea3eef79f10b6644d632